### PR TITLE
Portuguese translation

### PR DESCRIPTION
**Pull request by matheusfelipeog**
I did the dark theme translation 👍

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -4,12 +4,14 @@
   "title": {
     "en_EN": "Standard Dark Theme",
     "fr_FR": "Thème Noir Standard",
-    "sl_SI": "Standardna Temna Tema"
+    "sl_SI": "Standardna Temna Tema",
+    "pt_BR": "Tema Dark Padrão"
```
Comment (andersevenrud) [inline metadata.json:8]: Is "Dark" normal to use? :thinking:

**Comment by matheusfelipeog** [comment]
Finished, this is last 😄

**Review by andersevenrud — APPROVED** [review]
Looks good! Thanks :)

**Comment by andersevenrud** [comment]
Awesome! I've published these as well :)

File: metadata.json (hunk 1 above)
Comment (matheusfelipeog) [reply metadata.json:8]: Yes, the literal translation is strange

Comment (andersevenrud) [reply metadata.json:8]: I see :)

**Pull request merged**